types/constants.go

```diff
@@ -39,6 +45,11 @@ var (
 	// The GenesisID is used in many places. Calculating it once saves lots of
 	// redundant computation.
 	GenesisID BlockID
+
+	// Oak hardfork constants.
+	OakHardforkBlock     BlockHeight
+	OakDifficultyMaxRise = big.NewRat(1004, 1000)
+	OakDifficultyMaxDrop = big.NewRat(1000, 1004)
```

Comment:
do these belong in `types`? I thought previous hardfork values lived in `consensus`. I feel like they make more sense there.